themes/_core/js/product.js

```diff
@@ -171,7 +171,7 @@ function updateProduct(event, eventType, updateUrl) {
         $('.quickview .product-variants .page-product .product-variants').replaceWith(data.product_variants);
         $('.quickview .product-discounts, .page-product .product-discounts').replaceWith(data.product_discounts);
         $('.quickview .product-additional-info, .page-product .product-additional-info').replaceWith(data.product_additional_info);
-        $('.quickview #product-details, .page-product #product-details').replaceWith(data.product_details);
+        $('#product-details').replaceWith(data.product_details);
```

Comment:
But if the quickview also add a #product-details block, we are fucked :scream:

Reply:
It doesnt has it

Reply:
and it's ok if it change the modal product details, we are fucked if every product on product list has an id #product-details